src/main/java/org/springframework/social/salesforce/api/Salesforce.java

```diff
@@ -39,6 +39,8 @@
 
     public UserOperations userOperations();
 
+    public UserOperations userOperations(String gatewayUrl);
```

Comment:
What I am looking at is the pattern of how values are passed into the adapter from the SalesforceConnectionFactory.